### ISSUE TITLE
Inconsistent Results When Using Default 'From Date' and 'To Date' Values in Job Query Page

### ISSUE THREAD

**Issue by ashenr** [NAICNO/Jobanalyzer#507]
In the job query page, the default values for "From date" and "To date" are mentioned as "yesterday" and "today" respectively.

But when these values are typed in the input fields with the dates for yesterday and today, the results are different than they were set empty.

The results shown in the table should be the same when they are empty or when correct dates are set.

**Comment by lars-t-hansen** [comment]
Yeah... technically I think that the default for "from" is "24 h before now" which will be rounded down to the beginning of yesterday, and the default for "to" is "now" which will be rounded up to the beginning of tomorrow.  These are probably the correct defaults, all things considered, but I'm happy to discuss that.  Maybe the text for the "to date" should be something else, or maybe the rounding on an explicit "to date" is broken / should be something other than what it is.
